versioned_docs/version-2.5/explanations/integrations-in-rancher/logging/custom-resource-configuration/outputs-and-clusteroutputs.md

```diff
@@ -28,7 +28,7 @@ The `Output` is a namespaced resource, which means only a `Flow` within the same
 
```

Comment:
Sorry for the weird placement of this comment -- please add a space between the a id tag and the `##` markdown syntax on lines 22-23, because it's not rendering correctly in the preview.